xclim/__init__.py

```diff
@@ -1,11 +1,12 @@
 """Climate indices computation package based on Xarray."""
+from __future__ import annotations
+
 from importlib.resources import contents, path
 
 from xclim.core import units  # noqa
 from xclim.core.indicator import build_indicator_module_from_yaml
 from xclim.core.locales import load_locale
 from xclim.core.options import set_options  # noqa
-from xclim.indicators import atmos, land, seaIce  # noqa
```

Comment:
haha, what happened here? I think the unused imports remover was a bit insolent and forgot to listen to the `#noqa`? This one of the reason tests are failing. We need to import this to trigger the construction of all indices before being able to build the virtual ones (lower in this file).